bouncer/shared/gaslimit_ccm.ts

```diff
@@ -111,10 +106,10 @@ async function testGasLimitSwap(
 
   let gasSwapScheduledHandle;
 
-  if (CCM_CHAINS_NATIVE_ASSETS[destChain] !== sourceAsset) {
+  if (chainGasAsset(destChain as Chain) !== sourceAsset) {
     gasSwapScheduledHandle = observeSwapScheduled(
       sourceAsset,
-      destChain === 'Ethereum' ? Assets.Eth : ('ArbEth' as Asset),
+      destChain === 'Ethereum' ? Assets.Eth : Assets.ArbEth,
```

Comment:
just noticed there are lot of other places we still use asset as string, we should move to this Assets type. (ofc, would be another ticket/PR, not directly relevant)

Reply:
yes, there is definitely some cleanup to do in multiple places. I'll aim to get it done when I get some time.